scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
```

Comment:
about font sizes, I donot know about disadvantages of changing font size in media queries, but I  dothink using a media query there is unnecessary. A font size of 16/17px should be good for all screens, even 1080p displays.